templates/org/aac/aac-sample-solution-content.md

```diff
@@ -158,7 +158,7 @@ Other contributors: > Include contributing (but not primary) authors, major edit
  - [Contributor 2 Name](https://linkedin.com/in/ProfileURL) | Title, such as "Cloud Solution Architect"
```

Comment:
Can you catch the other ones on this page?  GitHub won't let me update lines that aren't near other changes.

```suggestion
 - [Contributor 2 Name](https://www.linkedin.com/in/ProfileURL/) | Title, such as "Cloud Solution Architect"
```